08-components-lab/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,47 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+import React from 'react';
+
+export default class TimeConverter extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {h: 0
+                     ,m:0
+                     ,s:0
+        };
+
+
+    }
+
+    handleChange(type, event)  {
+        switch(type){

```

Comment:
Instead of using multiple times ev.target.value you can assign it to a const in the beginning of a method. Alternative is to use ES6 destructuring in arrow function arguments. 
Consider this:

```
handleChange(type, { target: { value: seconds } }){
  this.setState({
     s: seconds
  })
}
```